.gitlab-ci.yml

```diff
@@ -14,6 +14,7 @@ Test:Required:
   before_script:
     - ''
   script:
+    - composer self-update
```

Comment:
Do you definitely want to always update composer? I can't see any reason not to. But I suppose there are some situations where it adds an ever so slight uncertainty to the process? Unless the security audit is still flagging in current builds, of course.